merrors/doc.go

```diff
@@ -19,12 +19,11 @@
 //
 // Example 3:
 //
-//	func CloseAll(closers []io.Closer) error {
-//		errs := merrors.New()
-//		for _ , c := range closers {
-//			errs.Add(c.Close())
-//    	}
-//		return errs.Err()
-//	}
-//
+//		func CloseAll(closers []io.Closer) error {
```

Comment:
You wanted to change that?

Reply:
So `make lint` fails on CI and points to `merrors/doc.go` being modified. But it passes locally. I then tried to format it via IDE, which changed it this way. Pushed to separate commit to see if this fixes it.

The same lint fail is on latest commit to main.

Reply:
🤔